src/main.cpp

```diff
@@ -73,47 +78,28 @@ extern "C" void app_main()
     initArduino();
     // Initialize NVS.
     auto nvs_err = nvs::begin();
-
     wifi::DTOConfig wifi_params;
-    wifi::DTOConfig wifi_params2;
-    setDefaultWiFiParams(wifi_params);
-
-
+    nvs::Namespace wifi_nvs;
+    wifi_nvs.open("wifi",nvs::OpenMode::ReadWrite);
```

Comment:
If you've been changed the `OpenMode` in `Namespace::open` to use nvs_open_mode you need to use `NVS_READWRITE` for type safety